app/src/app/[lng]/page.tsx

```diff
@@ -186,7 +190,7 @@ export default function Home({ params: { lng } }: { params: { lng: string } }) {
                   lineHeight="32"
                   fontWeight="semibold"
                 >
-                  Welcome Back,
+                  <Trans t={t}>welcome-back</Trans>,
```

Comment:
This is fine, but just FYI you don't necessarily have to use the Trans component in all places, this works too:
`{t("welcome-back")}`